README.md

```diff
@@ -5,31 +5,49 @@ Appwrite playground is a simple way to explore the Appwrite API & Appwrite Pytho
 **Work in progress**
 
 ## System Requirements
-* A system with Python3 installed.
-* You have readily available Appwrite running instance (localhost in most cases).
-* Create a project in Appwrite instance using console.
-* Generate a secret key in the Appwrite instance using console.
+* A system with Python 3+ or Docker installed.
+* An Appwrite instance.
+* An Appwrite project created in the console.
+* An Appwrite API key created in the console.
 
 ### Installation
 1. Clone this repository.
-2. cd into to repository.
-3. Open the app.py file found in the root of the cloned repository.
-4. Copy the project_id, endpoint, secret key from your Appwrite Console.
-4. Update project_id, endpoint, secret key by copied from the console in app.py file where stated.
-5. Install dependencies using pip `pip install appwrite`
-5. Execute the command `python app.py`
+2. `cd` into to repository.
+3. Open the playground.py file found in the root of the cloned repository.
+4. Copy Project ID, endpoint and API key from Appwrite console into `playground.py`
+5. Run the playground:
+    Python:
+        - Install dependencies using pip `pip install -r requirements.txt`
+        - Execute the command `python playground.py`
+    Docker:
+        - Execute the command `docker compose up`
 6. You will see the JSON response in the console.
 
-### API Covered in Playground.
-* Create Collection
-* List Collection
-* Add Document
-* List Documents
-* Upload File
-* List Files
-* Delete File
-* Create User
-* List User
+### API's Covered
+
+- Databse
+    * [Create Collection](./lib/playground.rb#L53)
+    * [List Collection](./lib/playground.rb#L111)
+    * [Add Document](./lib/playground.rb#L129)
+    * [List Documents](./lib/playground.rb#L148)
+    * [Delete Document](./lib/playground.rb#L157)
+    * [Delete Collection](./lib/playground.rb#L120)
+
+- Storage
+    * [Create Bucket](./lib/playground.rb#L169)
```

Comment:
Do we have list buckets? If not, can we add? Everything should have create/add/upload, list, and delete.